api/src/main/java/org/ehcache/config/CacheConfiguration.java

```diff
@@ -92,7 +92,7 @@
    * @deprecated Use {@link #getExpiryPolicy()}
    */
   @Deprecated
-  Expiry<? super K, ? super V> getExpiry();
+  org.ehcache.expiry.Expiry<? super K, ? super V> getExpiry();
```

Comment:
This is a separate commit. It fixes deprecation warnings caused by #2166. Here is need to use the full path because I can't suppress warnings on imports. A stupid Java quirks.

Reply:
But then where is the suppress warnings for that instance?

Reply:
It seems that since the method itself is deprecated, there's no need to suppress the warning